bin/create-user.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+USER_EMAIL="${1:-me@server.net}"
+FORENAME="${2:-John}"
+SURNAME="${3:-Smith}"
+PASSWORD=Password12
+USER_GROUP="${4:-bsp-systemupdate}"
```

Comment:
is this a user group that exists?

Reply:
yes `bsp-systemupdate` is configured in idam :)